client/my-sites/domains/domain-management/transfer/transfer-to-other-site/transfer-domain-to-other-site.tsx

```diff
@@ -55,7 +55,7 @@ export class TransferDomainToOtherSite extends Component< TransferDomainToOtherS
 		const isWpcomStagingSite = site?.is_wpcom_staging_site ?? false;
 
 		return (
-			site.capabilities.manage_options &&
+			site?.capabilities?.manage_options &&
```

Comment:
Not necessary for this PR but in a follow-up maybe we should alter [the type definition](https://github.com/Automattic/wp-calypso/blob/2616b8da2c63ca8b63582b9a3a7424d4429aecb5/client/my-sites/domains/domain-management/transfer/transfer-to-other-site/types.ts#L17) to make sure it's clear that [`selectedSite`](https://github.com/Automattic/wp-calypso/blob/2616b8da2c63ca8b63582b9a3a7424d4429aecb5/client/my-sites/domains/domain-management/transfer/transfer-to-other-site/types.ts#L25) and `capabilities` might not be set.

Reply:
Handled in: https://github.com/Automattic/wp-calypso/pull/90513